cdap-docs/admin-manual/source/installation/installation-note.txt

```diff
@@ -0,0 +1,18 @@
+.. topic:: Different ways to install CDAP
+
+  There are different ways to install CDAP, depending if it is a distributed or standalone system:
```

Comment:
it depends on much more than just distributed or standalone.  I would just say "depending on the underlying cluster setup" or similar